dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -1154,8 +1154,10 @@ protected void processMap(Contentlet contentlet, Map<String,Object> map) throws
 													contentlet.setHost(hh.getIdentifier());
 													contentlet.setFolder(folder.getInode());
 													if(st.getStructureType()==BaseContentType.FILEASSET.getType()){
-														String fileName = contentlet.getMap().get("fileName").toString();
-														Identifier existingIdent = APILocator.getIdentifierAPI().find(hh,split[1]+fileName);
+													    StringBuilder URI = new StringBuilder();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)